sources/packages/backend/workflow/src/workflow-definitions/parttime-configure-disbursement.bpmn

```diff
@@ -108,7 +108,9 @@
+          <zeebe:output source="={&#10;  CSLPAmount: round half up(finalFederalAwardNetCSLPAmount / 2, 0),&#10;  CSGPAmount: round half up(finalFederalAwardNetCSGPAmount / 2, 0),&#10;  CSGDAmount: round half up(finalFederalAwardNetCSGDAmount / 2, 0),&#10;  CSPTAmount: round half up(finalFederalAwardNetCSPTAmount / 2, 0),&#10;  BCAGAmount: round half up(finalProvincialAwardNetBCAGAmount / 2, 0),&#10;  SBSDAmount: round half up(finalProvincialAwardNetSBSDAmount / 2, 0)&#10;}" target="firstHalfDisbursementValues" />
```

Comment:
Was there a reason to provide the second parameter as zero? It seems to be the default behavior already unless I am missing something.
```
round half up(finalFederalAwardNetCSLPAmount / 2, 0)
```

Reply:
According to the documentation [link](https://docs.camunda.io/docs/components/modeler/feel/builtin-functions/feel-built-in-functions-numeric/#round-half-upn-scale), 0 is used to round to the ones. The calculation will give `null` value for using `round half up` without the scale parameter: `round half up(finalFederalAwardNetCSLPAmount / 2)` --> `null`.